phenopacketlab-restapi/pom.xml

```diff
@@ -100,6 +100,11 @@
             <groupId>com.google.cloud</groupId>
             <artifactId>spring-cloud-gcp-data-datastore</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I see that we're using `JSONObject` from this library in `org.monarchinitiative.phenopacketlab.restapi.domain.Phenopacket`.

Do you think we can use `com.fasterxml.jackson.databind.JsonNode` instead?
It seems to me (please let me know if I'm wrong) that we're using `JSONObject` just as a mean to go from JSON string to Phenopacket and back. Are there any other benefits I'm not aware of?

If not, I'd propose replacing `JSONObject` with `JsonNode` or just with `String` (or `byte[]`? :smiling_imp: ) and removing this dependency to keep things simpler.

This would likely result in needing to change
```java
public Map<String, Object> getPhenopacket() {...};
```
to 
```java
/**
 * @returns phenopacket as JSON string.
 */
public String getPhenopacket() {...};
```
that would be persisted.
@iimpulse @belkassaby do you think this would work with the datastore?